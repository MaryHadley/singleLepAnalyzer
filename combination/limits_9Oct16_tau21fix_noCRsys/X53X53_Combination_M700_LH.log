000 (1297366.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297366.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.235:42190>
...
006 (1297366.000.000) 10/10 19:27:26 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2268  -  ResidentSetSize of job (KB)
...
006 (1297366.000.000) 10/10 19:32:26 Image size of job updated: 283420
	159  -  MemoryUsage of job (MB)
	162116  -  ResidentSetSize of job (KB)
...
006 (1297366.000.000) 10/10 20:17:30 Image size of job updated: 284232
	160  -  MemoryUsage of job (MB)
	163020  -  ResidentSetSize of job (KB)
...
005 (1297366.000.000) 10/10 20:19:09 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:51:31, Sys 0 00:00:03  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:51:31, Sys 0 00:00:03  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1629  -  Run Bytes Sent By Job
	322  -  Run Bytes Received By Job
	1629  -  Total Bytes Sent By Job
	322  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1095        1   3824497
	   Memory (MB)          :      160     3072      3072
...
