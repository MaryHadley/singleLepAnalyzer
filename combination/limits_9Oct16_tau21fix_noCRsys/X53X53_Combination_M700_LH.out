/var/lib/condor/execute/dir_17735
Total number of likelihood evaluations: 55000099
Total number of likelihood evaluations: 5500099
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M700_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M700_LH/quant-toys:0-sig-d96b3a6f2c.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M700_LH/quant-data-sig-8289889ca5.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M700_LH
