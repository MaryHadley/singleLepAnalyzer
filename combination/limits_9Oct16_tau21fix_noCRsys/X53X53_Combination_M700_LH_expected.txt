# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0    0.13239   0.066397    0.28069   0.09356     0.1972
